FalmouthFalcons/iteration-2
# Bpw order controller test

Description:
-This added test file  for order controller

Number of Fixes:
-fixed fail in order model test
Related Ticket(s):
- List related ticket(s) that apply to the work done in this pull request
#5, #6 

Problem to Solve:
- Finish order controller

Proposed Changes:
-none
Expected Behavior:
- model tests should run
Steps to Test Solution
1) git pull origin master
2) git fetch
3) git checkout bpw-order-controller-test
4) ruby order_test.rb

Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

Deploy Notes:
- Notes regarding deployment the contained body of work.
- These should note any db migrations, etc.
